sdk/cosmos/azure-spring-data-cosmos-core/src/main/java/com/azure/spring/data/cosmos/repository/query/CosmosQueryMethod.java

```diff
@@ -64,18 +65,13 @@ public boolean hasAnnotatedQuery() {
      */
     @Nullable
     public String getQueryAnnotation() {
-        return findAnnotatedQuery().orElse(null);
+        return annotatedQueryValue;
     }
 
-    private Optional<String> findAnnotatedQuery() {
-
-        return lookupQueryAnnotation()
+    private Optional<String> findAnnotatedQuery(Method method) {
```

Comment:
Can we just delete method `findAnnotatedQuery`, and write the code directory in constructor?

Reply:
Good to have it in a function to keep the constructor clean